libs/wire-api/test/unit/Main.hs

```diff
@@ -14,18 +14,21 @@
 --
 -- You should have received a copy of the GNU Affero General Public License along
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
+{-# OPTIONS_GHC -Wno-unused-imports #-}
```

Comment:
This shouldn't be required, right?

Reply:
Ah, thanks, good catch. I just added it while trying some things out.